src/settings/utils.js

```diff
@@ -1,13 +1,23 @@
+import { Checkbox } from '@folio/stripes/components';
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 
 const NAME_REGEXP = new RegExp(/[^a-zA-Z\d]|^.{8,}$/);
 
-export const validatePrefixSuffixName = ({ name }) => ({
-  name: name?.match(NAME_REGEXP)
+export const validatePrefixSuffixName = (props) => {
+  const errors = {};
+  const name = props?.name?.match(NAME_REGEXP)
     ? <FormattedMessage id="ui-orders.settings.poNumber.nameValidation" />
-    : undefined,
-});
+    : undefined;
+  if(name != undefined) {
```

Comment:
we usually separate const and return statements with empty line, plz look at other utils in this file